utils/cloudfront.go

```diff
@@ -275,7 +274,7 @@ func (d *Distribution) SetCertificate(distId, certId string) error {
 	DistributionConfig.ViewerCertificate.IAMCertificateId = aws.String(certId)
 	DistributionConfig.ViewerCertificate.CertificateSource = aws.String("iam")
 	DistributionConfig.ViewerCertificate.SSLSupportMethod = aws.String("sni-only")
-	DistributionConfig.ViewerCertificate.MinimumProtocolVersion = aws.String("TLSv1")
+	DistributionConfig.ViewerCertificate.MinimumProtocolVersion = aws.String("TLSv1.1_2016")
```

Comment:
Noted this in Slack, but we probably don't want to cut off TLSv1.0 in this update without customer comms.